### PR TITLE
UserService_DeleteUserMassTransitAdding

### PR DESCRIPTION
**Pull request by dTarasova**

### CODE REVIEW

**Review by Cerberous007 — APPROVED** [review]
Maybe I missed something, but ovaerall is ok.

**Review by nmamro — COMMENTED** [review]
Seems fine overall, have you tested it?

File: Services/UserService/Validators/DeleteUserRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class DeleteUserRequestValidator: AbstractValidator<DeleteUserRequest>
+    {
+        public DeleteUserRequestValidator()
+        {
+            RuleFor(request => request.UserId)
+                .NotNull()
```
Comment (ifelseelif) [inline Services/UserService/Validators/DeleteUserRequestValidator.cs:11]: NotEmpty ensures that the object isn't null.

**Review by ifelseelif — APPROVED** [review]

**Review by nmamro — APPROVED** [review]

File: Services/UserService/Validators/DeleteUserRequestValidator.cs (hunk 1 above)
Comment (dTarasova) [reply Services/UserService/Validators/DeleteUserRequestValidator.cs:11]: > Seems fine overall, have you tested it?
Yeah, it's okay

**Review by VLiamin — APPROVED** [review]

**Pull request merged**